app/Repositories/ToDo/ToDoResponsitory.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Repositories\ToDo;
+
+use App\Interfaces\ToDo\ToDoRepositoryInterface;
+use App\Models\ToDo;
+use App\Repositories\BaseRepository;
+
+class ToDoResponsitory extends BaseRepository implements ToDoRepositoryInterface
+{
+    public function __construct(ToDo $toDo)
+    {
+        $this->model = $toDo;
+    }
+
+    public function findBydata($colum, $data, $operator = '=')
+    {
+        return $this->model->where($colum, $operator, $data)->orderBy('start_time', 'ASC')->get();
+    }
+
+    public function getByQuery($search = null, $where = [], $orderBy = [])
+    {
+        $result = $this->model;
+        $dataSet = $result->where(function ($query) use ($search) {
```

Comment:
chỗ này sao không dùng **$this->model** vào luôn mà phải tạo ra 1 biến $result vậy em ?